Makefile

```diff
@@ -51,6 +51,6 @@ nuke:
 regenerate:
 	make -C protoc-gen-go/descriptor regenerate
 	make -C protoc-gen-go/plugin regenerate
-	make -C protoc-gen-go/testdata regenerate
+	make -C protoc-gen-go/testdata buildprotos
```

Comment:
actually, I don't  know why that target is named differently. Can you revert this change and fix `protoc-gen-go/testdata/Makefile` instead?